docs/public/_redirects

```diff
@@ -8,6 +8,7 @@
 /r/x-license-eula https://mui.com/store/legal/mui-x-eula/ 302
 /r/x-license-key-installation https://mui.com/x/advanced-components/#license-key-installation 302
 /r/x-data-grid-no-dimensions https://mui.com/x/react-data-grid/layout/ 302
+/r/x-issue-template https://codesandbox.io/s/mui-x-issue-template-uh3pm2 302
```

Comment:
Please move the codesandbox under the MUI organization, so others can modify it later on.

<img width="199" alt="Screenshot 2022-07-09 at 11 43 53" src="https://user-images.githubusercontent.com/3165635/178100485-7b6b4ea6-c441-48bc-b571-42fac5ee82da.png">

Reply:
For example, we likely want to fix:

<img width="539" alt="Screenshot 2022-07-09 at 11 46 47" src="https://user-images.githubusercontent.com/3165635/178100581-7b62d0dc-cfa1-4e15-932a-9ac206b1c669.png">